tt_metal/include/compute_kernel_api.h

```diff
@@ -220,6 +220,11 @@ ALWI void abs_tile(uint32_t idst) { MATH((llk_math_eltwise_unary_sfpu_abs<APPROX
  */
 ALWI void abs_tile_init() { MATH((llk_math_eltwise_unary_sfpu_abs_init<APPROX>())); }
 
+/**
+ * Please refer to documentation for any_init.
```

Comment:
This description is for `_init` functions, not for the actual compute function. You can instead copy the description for `abs_tile()` (lines 202-215) and just make note that this version of the function is for int32 datums.